sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -48,7 +48,7 @@ export type SearchIndexClientOptions = PipelineOptions;
 /**
  * Class to perform operations to manage
  * (create, update, list/delete)
- * indexes, & synonymmaps.
```

Comment:
this looks like a too-eager find/replace :)

Reply:
Resolved the issue